mod_reforged/hooks/skills/actives/split_shield.nut

```diff
@@ -3,13 +3,14 @@
 	{
 		local ret = this.skill.getDefaultUtilityTooltip();
 
-		if (this.getShieldDamage(this.getContainer().getActor()) != 0)
+		local shieldDamage = this._RF_getShieldDamage();
+		if (shieldDamage != 0)
 		{
 			ret.push({
 				id = 7,
 				type = "text",
 				icon = "ui/icons/shield_damage.png",
-				text = "Inflicts " + ::MSU.Text.colorGreen(this.getShieldDamage(this.getContainer().getActor())) + " damage to shields"
+				text = format("Inflicts%s damage to shields", ::MSU.isNull(this.getContainer()) || this.getContainer().getActor().getID() == ::MSU.getDummyPlayer().getID() ? "" : " " + ::MSU.Text.colorRed(shieldDamage))
```

Comment:
You are missing a check for `::MSU.isNull(this.getContainer().getActor())` here before you access `getID`.
At least in the `RF_getShieldDamage` you added such a check.

Reply:
Vanilla sometimes seems to have that check and sometimes it doesn't. I personally think that check is unnecessary as an item_container should be expected to have an actor. I am not aware of a case where an is present in a container that doesn't have an actor, at least in the expected flow of the game. We can probably get away with removing that check from the other functions too, I guess I was just being "over safe" with it there.

Reply:
I also like removing that check and seeing how it goes under the assumption that is really never happens.